client/internal/wgproxy/proxy_userspace.go

```diff
@@ -35,7 +36,11 @@ func (p *WGUserSpaceProxy) AddTurnConn(turnConn net.Conn) (net.Addr, error) {
 	p.remoteConn = turnConn
 
 	var err error
-	p.localConn, err = nbnet.NewDialer().DialContext(p.ctx, "udp", fmt.Sprintf(":%d", p.localWGListenPort))
+	if runtime.GOOS == "ios" || runtime.GOOS == "android" {
```

Comment:
This should be android only, iOS already calls net.Dial underneath anyway. I'm not sure about Android either, it likely has `127.0.0.0/8` in the `local` table as well.